main.cpp

```diff
@@ -51,107 +51,115 @@ arm.flex();                             // 3)
 arm.rotateCounterClockwise();           // 4) demonstrates camelCase
 
 
-//  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
-//  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  1) 
+//  Noun: cat
+//  action 1: the cat purrs
+    cat.purr()
+//  action 2: the cat stretches
+    cat.stretch();
+//  action 3: the cat chases a ball
+    cat.chaseBall
+
+//  2) 
+//  Noun: lightning
+//  action 1: the lightning strikes the ground
+    lightning.strikeGround();
+//  action 2: the lightning strikes a tree    
+    lightning.strikeTree();
+//  action 3: the lightning dissapates  
+    lightning.dissapate();
 
 //  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
-//  4)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: puck
+//  action 1: the puck bounces
+    puck.bounce();
+//  action 2: the puck slides
+    puck.slide();
+//  action 3: the puck stops
+    puck.stop();
+
+//  4) 
+//  Noun: phone
+//  action 1: the phone rings
+    phone.ring();
+//  action 2: the phone vibrates
+    phone.vibrate();
+//  action 3: the phone lights up
+    phone.illuminate();
 
 //  5)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: bobblehead
+//  action 1: the bobblehead bobbles
+    bobblehead.bobble();
+//  action 2: the bobblehead wobbles
+    bobblehead.wobble();
+//  action 3: the bobblehead falls over
+    bobblehead.fallOver();
 
 //  6)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: oven
+//  action 1: the oven preheats
+    oven.preheat();
+//  action 2: the oven smokes
+    oven.emitSmoke();
+//  action 3: the oven cools
+    oven.coolDown();
 
 //  7)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: slot machine
+//  action 1: the slot machine reels spin
+    slotMachineReels.spin();
+//  action 2: the slot machine reels stop
+    slotMachineReels.stop();
+//  action 3: the slot machine bell rings
+    slotMachineBell.ring();
 
 //  8)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
-//  9)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: rice krispies
+//  action 1: the rice krispies snap
+    riceKrispies.snap();
+//  action 2: the rice krispies crackle
+    riceKrispies.crackle();
+//  action 3: the rice krispies pop
+    riceKrispies.pop();
+
+//  9) 
+//  Noun: 3d printer
+//  action 1: the 3d printer moves along the X axis
+    3dPrinter.moveX();
+//  action 2: the 3d printer auto-levels
+    3dPrinter.autoLevel();
+//  action 3: the 3d printer shuts off
+    3dPrinter.off();
 
 //  10)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
-
-
+//  Noun: programmer
+//  action 1: the programmer writes code
+    programmer.writeCode();
+//  action 2: the programmer fixes their code
+    programmer.fixBug();
+//  action 3: the programmer retires
+    programmer.retire();
```

Comment:
As explained in the instructions:
Here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
REPLACE the highlighted noun/action as follows:
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs `camelCase` for the name
- [ ] pick actions that need `camelCase` for their name.

Do not create an 11th noun.

Once you have completed the above-requested changes, please do the following:
- click each checkbox to mark each requested change as completed.
- click the `[Resolve Conversation]` button.
- Re-request a review of your pull request. It's on the right side of the Pull request page where you originally requested my review.
- Send me the link to your pull request in a new message.

Reply:
Done.